docs-gen/content/rule_set/data_entry/sensor_actuator.md

```diff
@@ -14,38 +14,64 @@ entry example is given below:
 ```YAML
 Speed:
   type: sensor
+  identifier: Vehicle.Speed
+  elementType: SignalDefinition
+  featureOfInterest: Vehicle
+  property: Speed
   description: The vehicle speed.
+  definition: The rate of change in the vehicle position per unit of time
   comment: For engine speed see Vehicle.Powertrain.CombustionEngine.Engine.Speed.
   datatype: float
   unit: km/h
   min: 0
   max: 300
 ```
 
-**```Drivetrain.Transmission.Speed```**
+**```Speed```**
 Defines the dot-notated name of the data entry. Please note that
 all parental branches included in the name must be defined as well.
 
 **```type```**
-Defines the type of the node. This can be `branch`,
-`sensor`, `actuator` or `attribute`.
+Defines the type of the node.
 
-**```datatype```**
-The string value of the type specifies the scalar type of the data entry
-value. See [data type](/vehicle_signal_specification/rule_set/data_entry/data_types/) chapter for a list of available types.
+**```identifier```** *[optional]*
+A set of characters that uniquely identifies the signal.  This is generally the branch and the property.
+
+**```elementType```** *[optional]*
+A type that classifies the attribute in regards to the Vehicle Signal domain
+- ```SignalDefinition```: The entry defines a property of a feature of interest.
+
+**```featureOfInterest```** *[optional]*
+The identifier of the physical object whose properties can be observed and possibly manipulated by signals of this type
+
+**```property```** *[optional]*
+The identifier of the property being reported by signals of this type
```

Comment:
If we are to introduce "optional" support for property while keeping the current mechanism of explicitly specifying datatype+unit for each signal we need to give some guidelines on how they shall be used. Topics like:

- Are they allowed/recommended for the standard catalog (i.e. the *.vspec files in this repo)
- If allowed/recommended in the standard catalog, is it then mandatory that the referenced property has the same datatype and unit as explicitly stated for the signal (to avoid confusion)
- If used for customization or extension, and datatype/unit differs between property and what is explicitly stated for signal, then what has priority?

Reply:
I'll add those, thanks.